core/src/main/java/com/capitalone/dashboard/model/ServiceStatus.java

```diff
@@ -4,14 +4,36 @@
  * Denotes the status of a product or service
  */
 public enum ServiceStatus {
-    Ok, Warning, Alert;
+    Ok(200, 200),
+    Warning(300, 400),
+    Unauth(401, 401),
```

Comment:
@brennanananan  - which case I see 401 error, can you put one scenario. I am trying to recreate the UNAUTH case and no luck !!.